Automattic/wp-calypso
# Site Settings: create a Back navigation button and add it to the JP Disconnect Site

**Purpose:** add an option of site Return (`Back button`) to the Jetpack Disconnect Flow.

**Included in the patch:**
- create a `NavigationBackButton` component, where the destination route is passed as a prop.
- add this component to the Jetpack Disconnect Site to have an option of Return to the starting point of the Disconnect flow (`Manage Connection` link in `/settings/general`).

**To test**: 
  -  Checkout this branch, or use calypso.live
    https://calypso.live/?branch=update/site-settings-disconnect-site-add-return
- Go to `settings/disconnect-site/:site`

For Jetpack and non-Atomic sites:
  -  verify that the `Back` button is visible in the left top corner and redirects to ```/settings/manage-connection/:site```, where :site is the Jetpack site.

All other sites: have no access = no site visibility

**Note:** please ignore the site settings specific Placeholder used here. This patch does not deal with adjusting it to the Disconnect Flow ( coming up).
 
**Visuals**:
![screen shot 2017-09-29 at 11 00 52](https://user-images.githubusercontent.com/13561163/31011225-6902aa7e-a505-11e7-952a-137841a85adc.png)
